src/pages/ProductList/ProductList.js

```diff
@@ -1,8 +1,141 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons'
+import './ProductList.scss'
 
 const ProductList = () => {
-  return <h1>hello</h1>
+  const [dropBox, setDropBox] = useState(false)
+
+  return (
+    <>
+      <div className="productList">
+        <header className="productListHeader">
+          <h1 className="productListTitle">
+            사료
+            <sup className="totalQuantity">총 {20}개</sup>
+          </h1>
+
+          <div className="headerContentBox">
+            <p className="headerContent">
+              마음의 양식은 뭐니 뭐니 해도 책이죠.
+            </p>
+            <p className="headerContent">
+              그렇다면 먹는 것을 다루는 책은 얼마나 좋은 양식일까요?
+            </p>
+            <p className="headerContent">
+              맛있는 책들을 보면서 마음까지 배부르게 채워보세요.
+            </p>
+          </div>
+        </header>
+        <div className="filterBox">
+          <div className="catAndDog">
+            <button className="catBtn">고양이</button>
+            <button className="dogBtn">강아지</button>
+          </div>
+          <button
+            className="dropBox"
+            onClick={() => {
+              setDropBox(prev => {
+                prev = !prev
+                return prev
+              })
+            }}
+          >
+            추천순
+            <FontAwesomeIcon
+              className="dropBoxArrow"
+              icon={faChevronDown}
+              style={{ color: '#000000' }}
+            />
+          </button>
+        </div>
+        {dropBox === false ? null : (
```

Comment:
```suggestion
        {dropBox && (
```
결국 dropBox state 값에 따라서 조건부 렌더링을 하는 구조기 때문에
위와 같은 방법으로 구현해주세요!